src/message/gneiss-message.ads

```diff
@@ -12,18 +12,12 @@
 private with Gneiss_Internal.Message;
 
 generic
-   type Buffer_Index is range <>;
-   type Byte is (<>);
-   type Buffer is array (Buffer_Index range <>) of Byte;
-   First  : Buffer_Index;
-   Length : Buffer_Index;
+   type Message_Buffer is private;
+   Null_Buffer : Message_Buffer;
 package Gneiss.Message with
    SPARK_Mode
 is
-   pragma Compile_Time_Error (Byte'Size /= 8, "Byte size must be 8 bit");
-
-   --  Fixed size message buffer
-   subtype Message_Buffer is Buffer (First .. First + Length - 1);
+   --  pragma Compile_Time_Error (Message_Buffer'Size /= 128 * 8, "Buffer size must be 128 byte");
```

Comment:
Why is that commented out? Are we missing a Workarounds entry here?

Reply:
I couldn't remember at first what the problem was. Sometimes I have to comment this kind of line out to get more usable error messages in generics. And when testing it locally and on Genode I had no problems anymore enabling this line.
However as you can see in the CI test this error occurs when using GNAT Community 2019 on Linux. When compiling the same code with GNAT Pro 20 it works. It also works on Genode.

Reply:
We should create a workaround issue in Componolit/Workarounds with a target tag Community_2020 (which hopefully fixes the issue) and comment out the check again.

Reply:
Documented in https://github.com/Componolit/Workarounds/issues/18 and fixed in https://github.com/Componolit/gneiss/pull/141/commits/b872839a0bb921cbbe07c90949941f9c8083172d.